.gitignore

```diff
@@ -3,3 +3,4 @@ build
 cli
 *.sw*
 mochawesome-report
+*.js
```

Comment:
Note: This likely should have been more targeted since we already have several JS files (correctly) checked in and could potentially cause headaches if trying to include a file like `.eslintrc.js` in the future:
```
cypress/integration/frontend.spec.js
cypress/plugins/index.js
cypress/support/commands.js
cypress/support/index.js
parse-python.js
webpack.config.js
```
fortunately git will continue tracking changes to these existing files.